File: C/product_of_a_list.c
// A C program to find a product of a list
// Author: Saketh-Chandra

#include <stdio.h>

int main()
{
    int lst[]={10, 5, 16, 23, 54, 6};
    float product = 1,i;
    for(i=0;i<6;i++)
    {
        product=product*lst[i];
    }
    printf("%f",product); // Output: 5961600  
    return 0;
}

File: Python/product_of_a_list.py

# A Python program to find a product of a list
# Author: Saketh-Chandra
# We can do this in two ways.

# 1st using Iteration
lst = [10, 5, 16, 23, 54, 6]
product = 1
for i in lst:
    product = product * i
print(product)  # Output 5961600

# 2nd using NumPy
import numpy as np

product = np.prod(lst)
print(product)  # Output 5961600
